docs/config.md

```diff
@@ -1397,6 +1398,43 @@ Default: `auto` (DHT + IPNI)
 
 Type: `optionalString` (`null`/missing means the default)
 
+
+### `Routing.AcceleratedDHTClient`
+
+Utilizes an alternative DHT client that searches for and maintains more information
+about the network in exchange for being more performant.
```

Comment:
```suggestion
Utilizes an alternative DHT client that searches for and maintains more information
about the network in exchange for improved provide latency performance (upwards of 100x).
```
(I made up the number above - I can't remember the actual numbers.  Maybe they are in the changelog for when we first released this?)

Can we maybe expand this by a sentence or two?  Is there a talk or doc we can link to?

Maybe something along the lines of...

```
This accelerated DHT client builds up a bamp of various nodes that can satisfy the keyspace.  This helps both with reads and writes:
* Reads - Traversed DHT results have been cached and know the closest nodes to query.
* Writes - In addition to having the cached traversed DHT results to know the direct n=20 nodes to store records with, records are batched so that do a single write with a large batch of records for a given provider.
```

(I know that's not a great writeup - but would like to have some more text that helps someone conceptually understand what is going on, or provide a link where someone can learn more so it's more self-service to reason about the system.)

Reply:
I guess some of these details are covered below in the "when it is enabled" section.  I mostly just want to make sure we help explain the concept of what's happening.

Reply:
I'll rework that section in the docs, I don't think changelogs should go in that much details, just link to docs that are that detailed.

Reply:
Btw the number is roughly 6 millions. 6 millions times faster.

Reply:
I belive that for all 20 provides, because almost all the time one of them will timeout and the timeout is 30s ?
Anyway 5µs is so small even if puts took 250ms this would still be way faster.